docs/elements.md

```diff
@@ -721,6 +723,52 @@ def generate(data):
 
 ---
 
+### `pl-units-input` element
+
+Fill in the blank field that allows for **numeric** input and accompanying **units**.
+
+#### Sample element
+
+**question.html**
+
+```html
+<pl-units-input answers-name="c_1" correct-answer="1m" atol="1cm"></pl-units-input>
+```
+
+#### Customizations
+
+| Attribute                  | Type                                         | Default      | Description                                                                                                                                                                                                                                                                             |
+| -------------------------- | -------------------------------------------- | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `answers-name`             | string                                       | &mdash;      | Variable name to store data in.                                                                                                                                                                                                                                                         |
+| `weight`                   | integer                                      | 1            | Weight to use when computing a weighted average score over elements.                                                                                                                                                                                                                    |
+| `correct-answer`           | string                                       | special      | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`.                                                                                                                                                                                                        |
```

Comment:
I think you are missing quotes around "answers-name" in the description field here.

Reply:
This is intentional (treating `answers-name` as a variable) and consistent with the docs for other elements.

Reply:
`answers-name` is not a valid python variable. It parses as an expression where you are subtracting the variable `answers` from the variable `name`

Reply:
I'm in favor of going back and addressing this later; all of the element docs do this (which I admit is kinda weird).